probe_scraper/runner.py

```diff
@@ -431,10 +428,24 @@ def load_glean_metrics(
         metrics = defaultdict(dict)
         pings = defaultdict(dict)
         for repo_name, commits in commits_by_repo.items():
+            this_repo = [r for r in all_repos if r.name == repo_name][0]
```

Comment:
this can retrun early if we use `next` instead:
```suggestion
            this_repo = next(r for r in all_repos if r.name == repo_name)
```